src/lib/zendesk/zendesk.client.ts

```diff
@@ -9,6 +9,77 @@ const zendeskClient = zendesk.createClient({
   proxy: process.env.http_proxy
 })
 
+export async function updateTicketWithStripeGoLiveResponse(ticket: ZendeskTicket, stripeGoLiveUrl: string, statementDescriptor: string, payoutStatementDescriptor: string): Promise<boolean> {
+  logger.info(`Updating zendesk ticket ${ticket.id} with go live response...`)
```

Comment:
```suggestion
  logger.info(`Updating zendesk ticket ${ticket.id} with go live response.`)
```

Reply:
Oh this ellipses is deliberate. I want the logs to look like:
```
Updating zendesk ticket XXX with go live response... 
Zendesk ticket XXX updated.
```

Reply:
Why do we need additional dots? Isn't the text clear

Reply:
The text is clear but ellipses can be used colloquially to indicate a pause. So it's like waiting for an event to happen. It's just a matter of personal preference though.

Reply:
Makes sense when we are waiting for things to happen in real-time or watching them happen but this ends up in logs which will be past. Don't think ellipses will convey a different meaning if someone is looking at logs.

Reply:
I don't think this is worth having a fight over so I've made the change to get this PR approved.